apps/internal/oauth/ops/authority/authority_test.go

```diff
@@ -317,3 +318,44 @@ func TestCreateAuthorityInfoFromAuthorityUri(t *testing.T) {
 		t.Errorf("TestCreateAuthorityInfoFromAuthorityUri: -want/+got:\n%s", diff)
 	}
 }
+
+func TestAuthParamsWithTenant(t *testing.T) {
+	uuid1 := "00000000-0000-0000-0000-000000000000"
+	uuid2 := strings.ReplaceAll(uuid1, "0", "1")
+	host := "https://localhost/"
+	for _, test := range []struct {
+		authority, expectedAuthority, tenant string
+		expectError                          bool
+	}{
+		{authority: host + "common", tenant: uuid1, expectedAuthority: host + uuid1},
+		{authority: host + "consumers", tenant: uuid1, expectedAuthority: host + uuid1},
+		{authority: host + "organizations", tenant: uuid1, expectedAuthority: host + uuid1},
+		{authority: host + uuid1, tenant: uuid2, expectedAuthority: host + uuid2},
+		{authority: host + uuid1, tenant: "common", expectError: true},
+		{authority: host + uuid1, tenant: "organizations", expectError: true},
+		{authority: host + uuid1, tenant: uuid2, expectedAuthority: host + uuid2},
```

Comment:
Did we mean to have this line here? Seems duplicated with line 333.

Reply:
Nope, good eye! I'll delete it.